src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -39,6 +50,16 @@ const (
 	defaultQemuMachineOptions = "accel=kvm,kernel_irqchip=on"
 
 	qmpMigrationWaitTimeout = 5 * time.Second
+
+	sevAttestationGrpcTimeout = 10 * time.Second
+
+	sevAttestationWorkingDir = "/opt/sev/"
```

Comment:
Why `/opt/` and not `/var/` for example?

Reply:
This is something I would really avoid hardcoding here.
Please, make it configurable via the configuration file as it most likely will change depending on the distro.

Reply:
@jimcadden, I'm adding a do-not-merge label till we get this part fixed here.  My other comments can be addressed in the future.

Reply:
I'm on it. Thanks @fidencio